packages/salesforcedx-vscode-apex/package.json

```diff
@@ -268,6 +268,11 @@
           "type": "boolean",
           "default": false,
           "description": "%enable_sobject_refresh_on_startup_description%"
+        },
+        "salesforcedx-vscode-apex.enable-completion-statistics": {
```

Comment:
@brpowell I could not find a way to have this be hidden from the settings UI. Do you happen to have more info on this?

Reply:
Could you talk a little bit about the reasoning why this need to be hidden from UI?

Reply:
I wouldn't say it needs to be hidden. It sounded like this was the preference during standup since we already have a lot of settings that users see on a regular basis. The only times we'd really want this to be enabled are 1) we are doing development and want to see the differences with what we make or 2) a customer is seeing an issue with completion and we ask them to enable this preference to get more info.

Reply:
Okay in that case I might add something like "debugOption" or "advanced" in the namespace: (So that it becomes `salesforcedx-vscode-apex.advanced.enable-completion-statistics`)

Similar to `salesforcedx-vscode-apex.java.home`
<img width="277" alt="Screen Shot 2020-05-29 at 7 06 53 PM" src="https://user-images.githubusercontent.com/679275/83317251-1c6eef80-a1e0-11ea-8e4d-4c0d62648887.png">

Reference:
- https://code.visualstudio.com/api/references/contribution-points#Configuration-schema - Search for "hierarchy".

Reply:
Yeah I was saying it might be a possible option. You might be able to get the configuration like you're doing so in the languageServer.ts file without this entry here but I'm not totally sure. As for whether it should be in the UI maybe @smaddox-sf should chime in.